design-system/materials/internals/definition.yaml

```diff
@@ -799,6 +868,8 @@ decisionGroupsByTheme:
           choice: shadow-0
         shadow-for-card-when-raised:
           choice: '0 1px 5px 0 rgba(0, 0, 0, 0.05)'
+        shadow-for-input-when-focused:
+          choice: none
```

Comment:
You might want to use the `shadow-0` choice token instead.

Reply:
Thanks @CarlosCortizasCT, Updated [4fd6a93](https://github.com/commercetools/ui-kit/pull/2347/commits/4fd6a934c68d96a095d68c7f4d60766a6590df94)